bridge/engine/EngineBarbell.h

```diff
@@ -33,6 +53,31 @@ struct EngineBarbell
     webrtc::WebRtcDataStream dataChannel;
     // some map for ssrc to user id endpointIdHash to be used in activemediaList, that we update from data channel
     // messages
+
+    struct SsrcGroup
+    {
+        uint32_t ssrcs[3];
+        uint32_t feedbackSsrcs[3];
+        uint32_t count = 3;
+
+        utils::Optional<size_t> endpointIdHash;
+        utils::Optional<std::string> endpointId;
```

Comment:
This string is disputable as we have potential mem alloc here in RT world.
When UMM is received over babrell data channel we will want to update this string and the idhash. It will be used later to create the UMM for clients.